src/main/resources/db/migration/V2024_04_01_1240__CIV-11300_hearing_scheduled_scenario.sql

```diff
@@ -0,0 +1,42 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.CP.Hearing.Scheduled.Claimant',
+        '{"Notice.AAA6.ClaimantIntent.GoToHearing.Claimant"}',
+        '{"Notice.AAA6.CP.Hearing.Scheduled.Claimant" : ["hearingDateEn", "hearingDateCy", "hearingCourtEn", "hearingCourtCy"]}'),
+       ('Scenario.AAA6.CP.Hearing.Scheduled.Defendant',
+        '{"Notice.AAA6.ClaimantIntent.GoToHearing.DefPartAdmit.Defendant", "Notice.AAA6.ClaimantIntent.GoToHearing.DefPartAdmit.FullDefence.StatesPaid.ClaimantConfirms.Defendant", "Notice.AAA6.ClaimantIntent.GoToHearing.DefPartAdmit.FullDefence.StatesPaid.PartOrFull.ClaimantDisputes.Defendant", "Notice.AAA6.ClaimantIntent.GoToHearing.DefFullDefence.ClaimantDisputes.Defendant", "Notice.AAA6.ClaimantIntent.GoToHearing.DefFullDefence.ClaimantDisputes.NoMediation.Defendant"}',
+        '{"Notice.AAA6.CP.Hearing.Scheduled.Defendant" : ["hearingDateEn", "hearingDateCy", "hearingCourtEn", "hearingCourtCy"]}');
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
```

Comment:
time_to_live needs to be inserted as 'Session' for notifications to be deleted after logout